samples/config.sample

```diff
@@ -0,0 +1,60 @@
+# camera-streaming-daemon configuration file is composed of sections,
+# each section has some keys and values. They
+# are case insensitive, so `Key=Value` is the same as `key=value`.
+#
+# [This-is-a-section]
+# ThisIsAKey=ThisIsAValuye
+#
+# Following specifications of expected sessions and key/values.
+#
+# Section [Mavlink]:
+#
+# Keys:
+#   Port
+#       MAVLink destination UDP port.
+#       Default: 14550
+#
+#   System_ID
+#       System ID of the Camera-Streaming-Daemon to be used in MAVLink
+#       communications.
+#       Default: 42
+#
+#   Component_ID
+#       Component ID of the Camera-Streaming-Daemon to be used in MAVLink
+#       communications.
+#       Default: 100 (MAV_COMP_ID_CAMERA)
+#
+#   Rtsp_Server_Addr
+#       IP address or hostname of the interface where the rtsp server is
+#       running. This is the address that will be used by the client to
+#       make the rtsp request.
+#       Default: 0.0.0.0
+#
+#   Broadcast_Addr
+#       Broadcast address to send MAVLink heartbeat messages.
+#       Default: 255.255.255.255
+#
+# Section [Gstreamer]:
```

Comment:
Daemon should have the capability to switch between multiple formats as per the service request instead of supporting only pre-defined types